mappings/net/minecraft/util/math/Box.mapping

```diff
@@ -1,10 +1,10 @@
 CLASS net/minecraft/class_646 net/minecraft/util/math/Box
-	FIELD field_2407 x1 D
-	FIELD field_2408 y1 D
-	FIELD field_2409 z1 D
-	FIELD field_2410 x2 D
-	FIELD field_2411 y2 D
-	FIELD field_2412 z2 D
+	FIELD field_2407 minX D
```

Comment:
why this change? are they not interchangable?

Reply:
makes more sense with `minX` and `maxX`